.maintain/node-template-release/src/main.rs

```diff
@@ -0,0 +1,278 @@
+use structopt::StructOpt;
+
+use std::{
+    collections::HashMap,
+    fs::{self, File, OpenOptions},
+    io::{Read, Write},
+    path::{Path, PathBuf},
+    process::Command,
+};
+
+use glob;
+
+use fs_extra::dir::{self, CopyOptions};
+
+use tempfile;
+
+use git2;
+
+use toml;
+
+use tar;
+
+use flate2::{write::GzEncoder, Compression};
+
+const FRONTIER_GIT_URL: &str = "https://github.com/paritytech/frontier.git";
+
+type CargoToml = HashMap<String, toml::Value>;
+
+#[derive(StructOpt)]
+struct Options {
+    /// The path to the `node-template` source.
```

Comment:
Indentation needs to be fixed. We use tabs in this repo.

Reply:
Just need to run a `rustfmt` I think?

Reply:
You can just do that, but make sure to create a rustfmt.toml file first and set hard_tabs to true.

Alternatively, most editors have support for the conversion. For example, `tabify` command in Emacs.

Reply:
I propose we put in the same one as substrate #431 , I can add that into this PR or another and rebase this one. (If you want to go that way)

Reply:
I'd say we just set `hard_tabes = true` and set everything else as default. There's not that much requirements on that and different formatting style will only lead to confusions.

Reply:
`cargo fmt -p node-template-release -- --config hard_tabs=true` run to resolve the formatting here.